src/shadowbox/scripts/update_mmdb.sh

```diff
@@ -15,7 +15,7 @@ for monthdelta in $(seq 10); do
     newdate=$(date --date="-$monthdelta months" +%Y-%m)
     ADDRESS="https://download.db-ip.com/free/dbip-country-lite-${newdate}.mmdb.gz"
     curl --fail --silent "${ADDRESS}" -o "$TMPDIR/$FILENAME.gz" > /dev/null && break  
-    if [[ $monthdelta -eq 10 ]]; then
+    if [ $monthdelta -eq 10 ]; then
```

Comment:
[The style guide is against `[..]`.](https://google.github.io/styleguide/shellguide.html#test----and---) This seems to be preferred:
```suggestion
    if (( monthdelta == 10 )); then
```

https://google.github.io/styleguide/shellguide.html#arithmetic

Reply:
`[[ ]]` and `(( ))` are bash-specific features.  This script's [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) is `#!/bin/sh`, so shellcheck requires it use only features that are available in standard POSIX shell.

This script runs inside the shadowbox container, which does not have bash.  (We install bash into the "build" container, but into the actual shadowbox container.)  This container uses the busybox shell implementation.  busybox's shell actually does provide `[[ ]]` (which is why this script worked), but in principle this script could fail if run on a system with a basic implementation of `/bin/sh`, even if that system also has bash installed at `/bin/bash`.

Apart from this change, we have a few other options:
* Add a `# shellcheck disable=...` pragma to silence the warning.
* Install bash in shadowbox (likely a few megabytes) and switch to `#!/bin/bash`.
* Change the shebang to `#!/bin/ash` (the `ash` shell, which is what busybox actually includes)
  - And also add a `# shellcheck ...` pragma to silence a warning that `ash` is not well-supported.  (There are many variants of `ash` so it's not clear which features a script can rely on.)